.circleci/config.yml

```diff
@@ -82,13 +89,13 @@ commands:
               staging)
                 DEPLOY_ENV=staging
               ;;
-              production)
+              prod*)
                 DEPLOY_ENV=prod
               ;;
               *)
                 DEPLOY_ENV=dev
                 BUILD_PARAMS=--sourceMap
-                echo "We are going to map << pipeline.git.branch >> to dev environment"
+                echo "Defaulting $DEPLOY_TARGET_ENV to dev environment"
               ;;
             esac
             echo "Setting deployment ENVIRONMENT to $DEPLOY_ENV"
```

Comment:
Modified the environment selection so that not only can be based on which branch triggered the build (this is the current setup) but that it can be specified via a pipeline parameter. The pipeline parameter with the target environment will be specified in command line script if we are doing a deploy.